mission_ctl.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/lightningnetwork/lnd/lnrpc"
+)
+
+func (r *regolancer) addFailedChan(fromStr string, toStr string, amount int64) {
+	r.mcCache[fromStr+toStr] = amount
+}
+
+func (r *regolancer) validateRoute(route *lnrpc.Route) error {
+	lastHopPK := ""
```

Comment:
I would call it previousHopPK, I found lastHop a bit confusing because I think of the last hop of the route

Reply:
:100: valid